deploy-service/common/pom.xml

```diff
@@ -112,7 +112,12 @@
         <dependency>
             <groupId>com.jayway.jsonpath</groupId>
             <artifactId>json-path</artifactId>
-            <version>0.9.1</version>
+            <version>2.9.0</version>
+        </dependency>
+        <dependency>
+            <groupId>net.minidev</groupId>
+            <artifactId>json-smart</artifactId>
+            <version>2.5.1</version>
```

Comment:
Any breaking changes that might impact existing functionality?

Reply:
this is only used for getting the database configuration from json  in DBConfigReader